functorch/_src/aot_autograd.py

```diff
@@ -1315,7 +1314,6 @@ class CompiledFunction(torch.autograd.Function):
         fw_metadata = _fw_metadata
 
         @staticmethod
-        @disable_torchdynamo
```

Comment:
why were we disabling dynamo earlier and not anymore?

Reply:
I moved the disable. Previously, we manually disabled dynamo on the forwards function we return. Now, dynamo is responsible for disabling itself on the returned compiled function. See https://github.com/pytorch/pytorch/pull/89736/files#diff-0b094ea719a9acd16c316b7ec6975f6c9825398952e9b3a67f180d20ac592d47R82

I wasn't really planning to do this change but import cycles were annoying